tools/tikv-control.md

```diff
@@ -19,6 +19,12 @@ TiKV Control (`tikv-ctl`) is a command line tool of TiKV, used to manage the clu
     $ tikv-ctl --ca-path ca.pem --cert-path client.pem --key-path client-key.pem --host 127.0.0.1:21060 <subcommands>
     ```
 
+    However sometimes `tikv-ctl` comunicates with `PD` instead of `TiKV`, in which case we need to use `--pd` option instead of `--host`. Here is an example:
```

Comment:
- Add a comma after "However"
- comunicates -> com**m**unicates
- we -> you
- use `--pd` option -> use the `--pd` option